test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -242,6 +242,32 @@ func (s *Suite) Define() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("should issue a certificate that defines a Common Name and Email Address", func() {
```

Comment:
What's the reason for including both common name *and* email SAN here? It will mean this test case won't run against issuers that don't support CNs but *do* support email SANs